test/topology/test_topology.py

```diff
@@ -112,6 +114,51 @@ async def test_replace_reuse_ip(manager: ManagerClient, random_tables) -> None:
     # TODO: check that group 0 no longer contains the replaced node (#12153)
 
 
+async def reconnect_driver(manager: ManagerClient) -> Session:
+    """Workaround for scylladb/python-driver#170:
```

Comment:
That issue was already fixed and our build toolchain uses driver released with this fix.

Reply:
> the driver persistently tries to connect to stopped node:

That's OK. It's not related to scylladb/python-driver#170 in any way.
Here you reconnect the driver basically to ignore the stopped nodes when creating a keyspace. scylladb/python-driver#170 was about driver not reconnecting after nodes were restarted (here you don't restart these stopped nodes).

> Anyway, maybe you know how to make it working in a more elegant way?

Just write an appropriate comment - explain why we need to reconnect the driver.

Anyway, this is part of a bigger story. I don't understand why the test works at all...

Reply:
added a comment

We are starting a completely new, unrelated cluster, `_seeds` in `ScyllaCluster` ignores stopped nodes.